run_FastQC_and_MultiQC.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -l
+# Author: Matilda Åslin (matilda.aslin@medsci.uu.se) & Monika Brandt (monika.brandt@medsci.uu.se)
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ]
+then
+  echo "" 
+  echo "This script will create a MultiQC-report for all fastq files in a given directory."
+  echo "It will search the directory given as input for fastq-files and then run FastQC on these file."
+  echo "MultiQC will be started automatically after FastQC is done."
+  echo ""
+  echo "Arguments and options:" 
+  echo "-h Display this text"
+  echo "-p Path to the Project directory in Unaligned, i.e. /<absolute path to runfolder>/Unaligned/<project>/. Required."
+  echo "-i INDEX If used, index-files will be included in the MultiQC report. Needed for chromium runs."
+  echo "-u INTEGER or several INTEGERs separated by comma. Lane numbers to be  included in report for Undetermined indices."
+  echo "   Will only work if fastq-files containing undetermined indices are located in Unaligned and if -p is pointing to /Unaligned/<project>/."
+  echo ""
+  echo "Example usage:"
+  echo ""
+  echo "bash run_FastQC_and_MultiQC.sh -p /full/path/to/runfolder/Unaligned/MyProject/ -i INDEX -u 2,4,6"
+  echo ""
+  exit 0
+fi
+
+while getopts p:i:u: option
+do
+case "${option}"
+in
+
+p) PROJECT_PATH=${OPTARG};;
+
+i) INCLUDE_INDEX=${OPTARG};;
```

Comment:
perhaps it's a bit more intuitive to just pass a flag instead of the flag + "INDEX"?
i.e. something like `i) INCLUDE_INDEX=true;;`

I'm not so familiar with `getops` though so I'm not 100% sure it will work like that